src/pconn.h

```diff
@@ -81,9 +81,9 @@ class IdleConnList: public hash_link, private IndependentRunner
     Comm::ConnectionPointer *theList_;
 
     /// Number of entries theList can currently hold without re-allocating (capacity).
-    int capacity_;
+    size_t capacity_ = 0;
     ///< Number of in-use entries in theList
-    int size_;
+    size_t size_ = 0;
```

Comment:
> yadij requested a review from rousskov

@yadij, you made changes to this PR and requested my review. A few days earlier, a new Squid contributor created #1929 with similar changes that I (partially) reviewed. We were making good progress with #1929 and revealed hidden [problems](https://github.com/squid-cache/squid/pull/1929#discussion_r1825048381) with some of the changes. AFAICT, those problems are now present in this PR as well. To avoid further duplication of effort (at least), please explicitly say which of the two PRs you want us to advance and/or close the PR that you do not want us to advance.